# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/fedcm/fedcm-disconnect-iframe.sub.https.html.ini
# Commit: da37e2d8a175
# Full Hash: da37e2d8a175fda08a98920d820dda0899b6aedf
# Author: Benjamin VanderSloot <bvandersloot@mozilla.com>
# Date: 2025-06-13 09:36:48
# Regressor Bug: 1969017
# File Overlap Count: 2
# Description:
#   Bug 1969017 - Refactor FedCM to use a managed actor rather than tagging onto WindowGlobalParent - r=anti-tracking-reviewers,emz
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D251617
# ==============================================================================

diff -r 15b92f98f62c -r da37e2d8a175 testing/web-platform/meta/fedcm/fedcm-disconnect-iframe.sub.https.html.ini
--- a/testing/web-platform/meta/fedcm/fedcm-disconnect-iframe.sub.https.html.ini	Thu Jun 12 18:16:52 2025 +0000
+++ b/testing/web-platform/meta/fedcm/fedcm-disconnect-iframe.sub.https.html.ini	Thu Jun 12 18:16:53 2025 +0000
@@ -1,18 +1,9 @@
 [fedcm-disconnect-iframe.sub.https.html]
-  expected:
-    if os == "android": OK
-    TIMEOUT
   [Same-origin iframe does not need explicit identity-credentials-get]
-    expected:
-      if os == "android": FAIL
-      TIMEOUT
+    expected: FAIL
 
   [Cross-origin iframe fails disconnect() without explicit identity-credentials-get]
-    expected:
-      if os == "android": FAIL
-      NOTRUN
+    expected: FAIL
 
   [Cross-origin iframe can disconnect with explicit identity-credentials-get]
-    expected:
-      if os == "android": FAIL
-      NOTRUN
+    expected: FAIL